app/views/compute_resources/form/_rackspace.html.erb

```diff
@@ -5,4 +5,4 @@
 <%= selectable_f(f, :region, f.object.regions, {}, {:label => _('Region'),
                  :help_inline => link_to_function(_("Test Connection"), "testConnection(this)",
                  :class => "btn + #{flavors.empty? ? "btn-default" : "btn-success"}",
-                 :'data-url' => test_connection_compute_resources_path) + image_tag('/assets/spinner.gif', :id => 'test_connection_indicator', :class => 'hide').html_safe }) %>
+                 :'data-url' => test_connection_compute_resources_path) + spinner('', :id => 'test_connection_indicator', :class => 'hide').html_safe }) %>
```

Comment:
does it make sense for spinner to add hide class by default? (or at least an option as pretty common to hide by default)?